### DISCUSSION TITLE
highlight shortcut not working

### DISCUSSION THREAD

**Discussion by hoeseidon** [sumatrapdfreader/sumatrapdf#4680] [Q&A]
i tried to add a shortcut for a different colored highlight but it failed

![image](https://github.com/user-attachments/assets/54776537-ae78-40dd-a087-489c08ec3a94)

**Comment by GitHubRulesOK** [comment]
The custom custom commands need current pre-release (Version  3.6+ candidate) and I think Name was supplemented by `ToolbarText` it will still work but less easy than one key or one icon

>ver 3.6+:: Name is optional. If given, the command will show up in command palette (Ctrl + K)
![image](https://github.com/user-attachments/assets/7b34257a-97a2-4407-b013-277c75a031d4)


```
Shortcuts [
	[
		Cmd = CmdCreateAnnotHighlight #eac4000 openedit
		Key = a
		Name = Ochre
		ToolbarText = OC
	]
]

```
![image](https://github.com/user-attachments/assets/29ec0ff4-c34b-4583-83a7-876300d14ded)
![image](https://github.com/user-attachments/assets/e6a0bce4-8d96-47a0-8ca2-2db38eb902aa)
